Makefile

```diff
@@ -194,17 +204,6 @@ publish-docs-staging:
 publish-docs-prod:
 	(cd ${ROOTDIR}; ./node_modules/.bin/ts-node documentation/scripts/documentation_publisher.ts push prod ${FLAGS})
 
-.PHONY: publish-docs-gh-pages
```

Comment:
FYI: Deleted this old GitHub Pages publishing target, since we stopped using it long ago.